src/VisualStudio/TestUtilities/Window/InteractiveWindow.cs

```diff
@@ -0,0 +1,147 @@
+            dteWindow.Close();
+
+            _interactiveWindowWrapper = _host.ExecuteOnHostProcess<InteractiveWindowWrapper>(typeof(RemotingHelper), nameof(RemotingHelper.CreateCSharpInteractiveWindowWrapper), (BindingFlags.Public | BindingFlags.Static));
+        }
+
+        public string LastReplOutput
+        {
+            get
+            {
+                var replText = ReplTextWithoutPrompt;
+                int lastPromptIndex = replText.LastIndexOf(ReplPromptText);
+
+                replText = replText.Substring(lastPromptIndex, (replText.Length - lastPromptIndex));
+                int lastSubmissionIndex = replText.LastIndexOf(ReplSubmissionText);
+
+                if (lastSubmissionIndex > 0)
+                {
+                    replText = replText.Substring(lastSubmissionIndex, (replText.Length - lastSubmissionIndex));
+                }
+                else if (!replText.StartsWith(ReplPromptText))
+                {
+                    return replText;
+                }
+
+                int firstNewLineIndex = replText.IndexOf(Environment.NewLine);
+
+                if (firstNewLineIndex <= 0)
+                {
+                    return replText;
+                }
+
+                firstNewLineIndex += Environment.NewLine.Length;
+                return replText.Substring(firstNewLineIndex, (replText.Length - firstNewLineIndex));
+            }
+        }
+
+        public string ReplText
+            => _interactiveWindowWrapper.CurrentSnapshotText;
+
+        public string ReplTextWithoutPrompt
+        {
+            get
+            {
+                var replText = ReplText;
+
+                // find last prompt and remove
+                int lastPromptIndex = replText.LastIndexOf(ReplPromptText);
+
+                if (lastPromptIndex > 0)
+                {
+                    replText = replText.Substring(0, lastPromptIndex);
+                }
+
+                // it's possible for the editor text to contain a trailing newline, remove it
+                return replText.EndsWith(Environment.NewLine) ? replText.Substring(0, (replText.Length - Environment.NewLine.Length)) : replText;
+            }
+        }
+
+        public bool CheckLastReplOutputEndsWith(string expectedText)
+            => LastReplOutput.EndsWith(expectedText);
+
+        public bool CheckLastReplOutputEquals(string expectedText)
+            => LastReplOutput.Equals(expectedText);
+
+        public bool CheckLastReplOutputContains(string expectedText)
+            => LastReplOutput.Contains(expectedText);
+
+        public void Reset(bool waitForPrompt = true)
+            => ResetAsync(waitForPrompt).ConfigureAwait(continueOnCapturedContext: false).GetAwaiter().GetResult();
+
+        public async Task ResetAsync(bool waitForPrompt = true)
+        {
+            await _host.ExecuteDteCommandAsync(DteReplResetCommand).ConfigureAwait(continueOnCapturedContext: false);
```

Comment:
Not explicitly specifying the argument here bugs me, because at first glance `ConfigureAwait(false)` looks like we are choosing to not configure await (I always no better, but still).